js/trickleView.js

```diff
@@ -70,15 +70,13 @@ define([
             }
 
             Adapt.trigger("trickle:steplock", this);
-            //console.log("trickle steplock at", this.model.get("_id"))
-
             this.isSteplocked = true;
         },
 
         continueToNext: function() {
-            _.defer(_.bind(function() {
+            _.defer(function() {
                 Adapt.trigger("trickle:continue", this);
-            }, this));
+            }.bind(this));
```

Comment:
could be changed to `_.defer(Adapt.trigger, "trickle:continue", this);`

Reply:
eehh, no, this context issue again.